contributors/design-proposals/auth/pod-restriction.md

```diff
@@ -0,0 +1,477 @@
+# PodRestrictions
+
+## Overview
+
+This is a proposal for a new policy object (and associated admission controller) that adds basic
+restrictions to most fields on a Pod. This is motivated by a desire to unify the various policies we
+have now (which all behave in slightly different ways), and to unblock expanding the set of
+restricted fields. Furthermore, it aims to solve some of the key usability problems with
+PodSecurityPolicies, such as the inability to compose policies and the difficulty in binding
+policies.
+
+### Existing Policies
+
+There are multiple policies that constrain pod objects today. PodRestriction would supersede these
+policies, and they would eventually be deprecated.
```

Comment:
The admission controllers mentioned here for the most part all perform mutation/defautling so how do you envision deprecating them in favor of this admission controller if it isn't supposed to mutate/default fields?

Reply:
That sounds like it may get complex for PSP's since that admission controller has very specific logic around choosing a PSP to apply to a pod. 

cc @liggitt